### DISCUSSION TITLE
INmulti def_outputs as bpatcher

### DISCUSSION THREAD

**Discussion by the-steech** [ppooll-dev/ppooll#82] [Ideas]
experimenting in the live.ppooll branch, i wanted a more reliable way to open the "outs" def menu (to play nicely as a bpatcher).  i decided to add it as a bpatcher to the top level of the act.  now i just change the window size to show/hide the outputs, and they stay attached to the actual act.

any thoughts on this?  i think it works a little nicer, so that you keep everything in the same act window.  it will also make more sense for live.ppooll

<img width="370" alt="image" src="https://user-images.githubusercontent.com/114934553/216719505-ce01623d-7b4d-4992-81e2-448f13b2b70c.png">

**Comment by klausfilip** [comment]
ok!
can you open-close the bpatcher then with act-window sizing?
i did not get, why this is more convenient, than an extra window..

**Comment by chausch** [comment]
not very much a fan of this i have to admit 😕

**Reply by klausfilip** [reply]
why?
what is the trouble with this?
sizing the act-window is maybe awkward sometimes.

**Comment by klausfilip** [comment]
btw, @the-steech  - facing your screenshot up here in the first post,
looks as if you still are a bit unclear about how this multichannel routing is working ;)
its only ho_st1::out.1 in the first line !
with the underscores in the 3 following lines, a 4-channel signal is sent to ho_st.
(yes your setting is doing nearly the same. but sending two separate signals:
a 1-channel signal to out.1  and a 3-channel signal to out.2)

**Comment by klausfilip** [comment]
and there is a pull-request into main.
i tested quite a bit and don't have crashes any more.
besides the priority sometimes, there have been [delay-defer] in some externals (like ll.s~ etc) that were causing crashes.
some externals derive from the max5 version, with bad habbits of sequencing the messages...
